templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -77,9 +77,15 @@ contents:
         echo "MTU found for iface: ${iface}: ${iface_mtu}"
       fi
 
-      # create bridge
+      # create bridge; use NM's ethernet device default route metric (100)
```

Comment:
I'm wondering if we should use the metric of the interface we take instead.

Reply:
Yes, we technically should. But even that doesn't solve the real problem; bonds default to 300. So if you have a bond as your default interface, set up OVS, and then hot-plug another NIC, you're still screwed.

The real fix is to make sure that when you hotplug a NIC that you don't want to be the default route, you somehow tell the guest OS that it shouldn't be the default route. THat's way out of scope for us here.

Reply:
if that's the argument, then shouldn't we make our route even lower? if someone hot plugs and gets the same metric coudln't we have the same problem? Maybe we should make it 99.

Reply:
@trozet NM dynamically adjusts default route priorities to make sure they don't conflict, and since we're hardcoding one, it will make sure other devices dont' conflict. So yeah, we could do 99, but don't need to. Since we've claimed 100, all other interfaces will be assigned higher priorites (eg >= 101)